cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,94 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('/webtables');
+  });
+
+  it('should test pagination', () => {
+    cy.get('.left-pannel').click();
```

Comment:
The test 'should test pagination' is not implemented. Currently it just clicks on an element with the class name '.left-pannel'. The test should check if the pagination works as expected.